### PR TITLE
Circuit sandbox with symbolic notation

### PR DESCRIPTION
**Pull request by korgan00**
## Changes

Fix https://github.com/Qiskit/platypus/issues/1083

## How to read this PR

[https://platypus-pr-1086.ecy0akwlcpw.us-south.codeengine.appdomain.cloud/course/examples/widgets](https://platypus-pr-1086.ecy0akwlcpw.us-south.codeengine.appdomain.cloud/course/examples/widgets)

### CODE REVIEW

File: frontend/vue/components/MiniComposer/CircuitSandboxWidget.vue [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+      <section class="mini-composer__state-views__view">
+        <h1 class="mini-composer__state-views__view__title">
+          Matrix
+        </h1>
+        <SymbolicNotation
+          class="mini-composer__state-views__view__area"
+          :tex="matrixTex"
+        />
+      </section>
+      <section class="mini-composer__state-views__view">
```
Comment (techtolentino) [inline frontend/vue/components/MiniComposer/CircuitSandboxWidget.vue:44]: @korgan00 - does this need a class? I don't see it being used. The title and area within the section _can_ be

`mini-composer__state-views__title`
`mini-composer__state-views__area`

File: frontend/vue/components/MiniComposer/CircuitSandboxWidget.vue [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+  }
+  &__state-views {
+    grid-area: lesson;
+    display: flex;
+    flex-direction: column;
+    gap: $spacing-05;
+    border-left: 1px solid $border-color;
+    padding: $spacing-05;
+
+    &__view {
```
Comment (techtolentino) [inline frontend/vue/components/MiniComposer/CircuitSandboxWidget.vue:257]: might not need this class -- what are your thoughts?

File: frontend/vue/components/MiniComposer/Gate.vue [hunk 3]
```diff
@@ -68,7 +67,7 @@ export default class Gate extends Vue.with(Props) {
 .gate {
```
Comment (techtolentino) [inline frontend/vue/components/MiniComposer/Gate.vue:67]: should we add a `hover` cursor, here? the same way we have one for the Eigenvector widget?

https://user-images.githubusercontent.com/6276074/173419592-94b44f2e-8d31-4844-b1b8-49164b406e99.mov

**Review by techtolentino — CHANGES_REQUESTED** [review]
@korgan00 - incredible work, thank you 👏 

I just left a few small comments.

File: frontend/vue/components/MiniComposer/CircuitSandboxWidget.vue [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+        }
+      }
+    }
+  }
+  &__state-views {
+    grid-area: lesson;
+    display: flex;
+    flex-direction: column;
+    gap: $spacing-05;
+    border-left: 1px solid $border-color;
```
Comment (techtolentino) [inline frontend/vue/components/MiniComposer/CircuitSandboxWidget.vue:254]: on mobile, I think we can hide this `border-left`
Screen Shot 2022-06-13 at 11 31 09 AM

Comment (korgan00) [reply frontend/vue/components/MiniComposer/CircuitSandboxWidget.vue:258]: Also added a border top :)

**Review by techtolentino — APPROVED** [review]
This is incredible @korgan00 

👏 

thanks for adding the Reset button, and for addressing my comments.

**Pull request merged**